packages/gcc_build.rb

```diff
@@ -140,18 +140,18 @@ def self.build
     Dir.chdir('objdir') do
       build_env =
         {
-      build_configargs: @gcc_global_opts,
-                    AR: 'gcc-ar',
-              BASH_ENV: "#{CREW_PREFIX}/etc/env.d/rust",
-                CFLAGS: @cflags,
-           CREW_LINKER: 'ld',
-              CXXFLAGS: @cxxflags,
-               LDFLAGS: @ldflags,
-                    LD: 'ld',
-          LIBRARY_PATH: "#{CREW_GLIBC_PREFIX}:#{CREW_LIB_PREFIX}",
-                    NM: 'gcc-nm',
-                  PATH: @path,
-                RANLIB: 'gcc-ranlib'
+                       build_configargs: @gcc_global_opts,
+                                     AR: 'gcc-ar',
+                                 CFLAGS: @cflags,
+                            CREW_LINKER: 'ld',
+      CREW_PRELOAD_ENABLE_COMPILE_HACKS: '1',
```

Comment:
This is already handled by `package.rb`:
https://github.com/chromebrew/chromebrew/blob/3ee0c821390abfd579e640f8481a4045404db700/lib/package.rb#L359-L361